staging/knative-serving/values.yaml

```diff
@@ -0,0 +1,1179 @@
+activator:
+  activator:
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      readOnlyRootFilesystem: true
+      runAsNonRoot: true
+      seccompProfile:
+        type: RuntimeDefault
+    env:
+      configLoggingName: config-logging
+      configObservabilityName: config-observability
+      gogc: "500"
+      metricsDomain: knative.dev/internal/serving
+    image:
+      repository: gcr.io/knative-releases/knative.dev/serving/cmd/activator
+      tag: eebd9aa725b7
+    resources:
+      limits:
+        cpu: "1"
+        memory: 600Mi
+      requests:
+        cpu: 300m
+        memory: 60Mi
+activatorService:
+  ports:
+  - name: http-metrics
+    port: 9090
+    targetPort: 9090
+  - name: http-profiling
+    port: 8008
+    targetPort: 8008
+  - name: http
+    port: 80
+    targetPort: 8012
+  - name: http2
+    port: 81
+    targetPort: 8013
+  - name: https
+    port: 443
+    targetPort: 8112
+  type: ClusterIP
+autoscaler:
+  autoscaler:
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      readOnlyRootFilesystem: true
+      runAsNonRoot: true
+      seccompProfile:
+        type: RuntimeDefault
+    env:
+      configLoggingName: config-logging
+      configObservabilityName: config-observability
+      metricsDomain: knative.dev/serving
+    image:
+      repository: gcr.io/knative-releases/knative.dev/serving/cmd/autoscaler
+      tag: 07920fd1f08b
```

Comment:
is `staging/knative-serving/values.yaml` copied directly with no modifications from upstream ? if yes, can you maybe paste a link to the exact SHA of the commit this file was copied from..I can imagine someone debugging a pod and will refer to this tag and it might not be trivial to reverse engineer where this tag is coming from, will also be helpful for future upgrades.

Reply:
@takirala This was addressing a previous fix for knative I saw a PR for. In it, all SHA based tags were converted to just the version tag, for air-gapped support. Basically I went to the repo, and matched the SHA to the corresponding tag, and changed the file

Reply:
![image](https://user-images.githubusercontent.com/22758527/233473600-e35bdbe5-6843-4b0d-8eb9-737b5d965d93.png)

Reply:
for example

PR for reference: https://github.com/mesosphere/charts/pull/617

Reply:
oh. Yes we can!